src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.html

```diff
@@ -219,14 +231,16 @@ <h5 class="h5-margin-top">
                         class="autocomplete-dropdown"
                         *ngFor="let cityName of cityPredictionList"
                         [value]="cityName.description"
-                        (click)="onCitySelected(i, cityName)"
+                        (onSelectionChange)="onCitySelected(i, cityName)"
                       >
                         {{ cityName.description }}
                       </mat-option>
                     </mat-autocomplete>
                     <div
-                      class="validation"
-                      *ngIf="address.get(getLangValue('city', 'cityEn')).invalid && address.get(getLangValue('city', 'cityEn')).touched"
+                      *ngIf="
+                        address.get(getLangValue('city', 'cityEn')).errors?.required ||
+                        address.get(getLangValue('city', 'cityEn')).errors?.requiredFromDropdown
+                      "
```

Comment:
code duplication, create a method with 2 parameters( city, cityEn)

Reply:
I added the function, but with 3 arguments `setValidationInputError(address, 'city, 'cityEn')`, because we have several addresses there)